JASP-Engine/JASP/R/regressionloglinearbayesian.R

```diff
@@ -15,811 +15,576 @@
-					
-					for (var in seq_along(coef)) {
-					
-						Bayesianlogregression.result[[ len.Blogreg ]] <- emptyRow
-						terms <- coef[[var]]
-						actualName<-list()
-					
-						for (j in seq_along(terms)){
-							actualName[[j]] <- paste(lookup.table[[ terms[j] ]], collapse=" = ")
-						}			
-						varName<-paste0(actualName, collapse="*")
-							
-						Bayesianlogregression.result[[ len.Blogreg ]]$"Name" <- varName
-						Bayesianlogregression.result[[ len.Blogreg ]]$"post_prob" <- as.numeric(logBlm.estimates$prob[var])
-						Bayesianlogregression.result[[ len.Blogreg ]]$"post_mean" <- as.numeric(logBlm.estimates$post_mean[var])
-						Bayesianlogregression.result[[ len.Blogreg ]]$"post_var" <- as.numeric(logBlm.estimates$post_var[var])
-						
-						if (options$regressionCoefficientsCredibleIntervals == TRUE){			
-							Bayesianlogregression.result[[ len.Blogreg ]]$"lower_lim" <- as.numeric(logBlm.estimates$lower[var])
-							Bayesianlogregression.result[[ len.Blogreg ]]$"upper_lim" <- as.numeric(logBlm.estimates$upper[var])
-						}
-					
-						len.Blogreg <- len.Blogreg + 1
-					}		
-				}			
+      len.Blogreg <- len.Blogreg + 1
+      
+      for (var in 1:length(variablesInModel)) {
+        
+        if (base::grepl(":", variablesInModel[var])) {
+          
+          # if interaction term					
+          vars <- unlist(strsplit(variablesInModel[var], split = ":"))
+          name <- paste0(vars, collapse = "\u2009\u273b\u2009")
+          
+        } else 
+          name <- as.character(variablesInModel[ var])
+        
+        results[[ len.Blogreg ]]$"Name" <- name
+        len.Blogreg <- len.Blogreg + 1
+      }
+    }
+  }
+  container[["SummaryTable"]]$addRows(results)
+}
 
-			} else {
-		
-				len.Blogreg <- length(Bayesianlogregression.result) + 1
-				Bayesianlogregression.result[[ len.Blogreg ]] <- dotted.line
-		
-				if (length(bfObject$variables) > 0) {
-			
-					variablesInModel <- bfObject$variables
-			
-					len.Blogreg <- len.Blogreg + 1
-			
-					for (var in 1:length(variablesInModel)) {
-				
-						Bayesianlogregression.result[[ len.Blogreg ]] <- dotted.line
-				
-						if (base::grepl(":", variablesInModel[var])) {
-					
-							# if interaction term					
-							vars <- unlist(strsplit(variablesInModel[var], split = ":"))
-							name <- paste0(vars, collapse="\u2009\u273b\u2009")
-					
-						} else {
-					
-							name <- as.character(variablesInModel[ var])
-						}
-				
-						Bayesianlogregression.result[[ len.Blogreg ]]$"Name" <- name
-						len.Blogreg <- len.Blogreg + 1
-					}
-				}
-			}
-		
-		} else {
-						
-			len.Blogreg <- length(Bayesianlogregression.result) + 1
+.basRegLogLinSubSummaryResults <- function(container, dataset, options) {
+  if(!is.null(container[["SubSummaryResults"]]) || !options$regressionCoefficientsSubmodel) 
+    return()
+  # Compute/get the model
+  bfObject <- .basRegLogLinComputeBFObject(container, dataset, options)
```

Comment:
Can you add a check here to see if the submodel specified is actually within 1:nModels?